larq/metrics.py

```diff
@@ -62,7 +62,49 @@ def get_training_metrics():
     return _GLOBAL_TRAINING_METRICS
 
 
-class FlipRatio(Metric):
+class LarqMetric(Metric):
+    """Metric with support for both 1.13 and 1.14+"""
+
+    def reset_states(self):
+        return tf.keras.backend.batch_set_value([(v, 0) for v in self.variables])
+
+    def add_weight(
+        self,
+        name,
+        shape=(),
+        aggregation=tf.VariableAggregation.NONE,
```

Comment:
I'd prefer not changing the default here, so that `LarqMetric` will behave the same as `keras.metrics.Metric`.
```suggestion
        aggregation=tf.VariableAggregation.SUM,
```